app/models/question.rb

```diff
@@ -0,0 +1,16 @@
+class Question < ActiveRecord::Base
+  include Scoring
+  validates :content, presence: true
+  validates :title, presence: true
+
+  belongs_to :user
```

Comment:
like i mentioned briefly yesterday.. if you have a `belongs_to` that is required - you probably want to add a validation on that foreign_key field to ensure that it is associated with a parent object before it's saved. we don't want bad data in our database